pokemon-react-app/package.json

```diff
@@ -3,15 +3,26 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@babel/core": "^7.13.0",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+    "axios": "^0.24.0",
+    "es6-promise": "^4.2.8",
```

Comment:
You have added here some libs but I don't see how you used them? Like es6-promise or
isomorphic-fetch. If they are just used to polyfill old browsers there is no need in this case.
What about react-color?